package_builder/genie_python_install.bat

```diff
@@ -39,18 +39,27 @@ if exist "%ZIPPROG%" (
 
 @echo %TIME% genie_python robocopy started
 if exist "%BASEDIR%\Python\ZIP_ONLY_INSTALL.txt" (
+    if not exist "%ZIPPROG%" (
+        @echo ERROR: No 7-ZIP program found for genie_python install
```

Comment:
Wouldn't it be better to fall back to a builtin like `tar` rather than just erroring in this case? I guess 7-z is faster than tar if it's installed?

Reply:
tar didn't used to be available on windows, it appear in some windows 10 version by the look of it. I though we couldn't fall back to it currently with  `7zip` archives, but it looks like our default compression is compatible. I though it wasn't but i'd picked up tar from somewhere else on my computer...

Reply:
though i'll need to check that happens if you use 7z filters on compression